### PR TITLE
Common procedures

### PR DESCRIPTION
**Pull request by Tonyynot14**
Made a view to bring information about common procedures including min,max,average and facilities that the procedures have occurred at

### CODE REVIEW

File: app/controllers/bills_controller.rb [hunk 1]
```diff
@@ -77,6 +81,13 @@ def destroy
     @bill.destroy
     redirect_to admins_path
   end
+  
+  def common
+    @countOfProcedure =Bill.group(:procedure_id).count
```
Comment (micronix) [inline app/controllers/bills_controller.rb]: use snake case naming convention for variable names.
```suggestion
    @count_of_procedure = Bill.group(:procedure_id).count
```

File: app/models/facility.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class Facility < ActiveRecord::Base
-    has_many :bills
+    has_many :bill
```
Comment (micronix) [inline app/models/facility.rb]: I think this should be plural `has_many :bills` is there a reason why you changed to singular?

File: app/views/bills/common.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<h4> Common Procedures </h4>
+
+<% @countOfProcedure.each do |procedure|%>
+  <div>
+  <h5>  <%= @procedures.find(procedure[0]).name %></h5> Occuried <%= procedure[1] %> times
+    <%currentBill = @bills.where(procedure_id: procedure[0])%>
```
Comment (micronix) [inline app/views/bills/common.html.erb]: use snake case naming convention for variables `current_bill`

**Review by micronix — CHANGES_REQUESTED** [review]
Just some style changes

File: app/models/facility.rb (hunk 2 above)
Comment (Tonyynot14) [reply app/models/facility.rb]: Complete accident

**Review by micronix — APPROVED** [review]
looks good

**Pull request merged**